views/_sphinx_static_files.html

```diff
@@ -1,87 +1,88 @@
 <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/vega-embed@3.14.0/build/vega-embed.js"></script>
-<!--<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/builtin.js" type="text/javascript"></script>-->
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/codemirror.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/runestonebase.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/activecode.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/mchoice.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/shortanswer.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/js_lib/prettify.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/js_lib/hammer.min.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/parsons.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/fitb.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/dragndrop.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/timeddnd.js" type="text/javascript"></script>
-<script src="/{{=request.application}}/books/published/{{=course['base_course']}}/_static/lp.js" type="text/javascript"></script>
+<!--<script src="{{=base_static_url}}/builtin.js" type="text/javascript"></script>-->
+<script src="{{=base_static_url}}/codemirror.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/runestonebase.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/activecode.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/mchoice.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/shortanswer.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/js_lib/prettify.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/js_lib/hammer.min.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/parsons.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/fitb.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/dragndrop.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/timeddnd.js" type="text/javascript"></script>
+<script src="{{=base_static_url}}/lp.js" type="text/javascript"></script>
```

Comment:
This `lp.js` file appears to be missing from the PR or else it might be a typo??

Reply:
Oops, thanks. That belongs in my next PR (adding literate programming). I'll delete it for now.